Update GCP BRoker Provider doc and add Azure Broker Docs (#2351)
kyma-project/kyma: docs/service-brokers/helm-broker-service-classes/gcp-broker/docs/overview.md

---
title: Overview
type: Overview
---

> >**NOTE:** You must create a Secret with provider credentials in every Namespace where you provision the GCP Service Broker class. You can provision only one instance of the GCP Service Broker in each Namespace.

Reviewer: ```suggestion
>**NOTE:** You can provision only one instance of the GCP Service Broker in each Namespace. In every Namespace, you must create a Secret with provider credentials. Read the following section to learn how.
```

Author: Let's discuss with PK (the order os sentences)

The GCP Service Broker class exposes services from the [Google Cloud Platform Service Broker](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker)
in a given Namespace of the Kyma cluster.

> ## How to create a secret

Reviewer: ```suggestion
## Create a Secret
```

Reviewer: The same question: isn't it documented somewhere so that we'd only link to the external sources?

Author: this document is removed

> To add GCP Service Broker to your Namespace, you must prepare a service account and a 

Reviewer: ```suggestion
To add the GCP Service Broker to your Namespace, you must prepare a service account and a 
```
